lib/cli/ui/spinner/spin_group.rb

```diff
@@ -81,9 +81,10 @@ def check
           #
           # * +index+ - index of the task
           # * +force+ - force rerender of the task
+          # * +width+ - current terminal width to format for
           #
-          def render(index, force = true)
-            return full_render(index) if force || @force_full_render
+          def render(index, force = true, width: CLI::UI::Terminal.width)
```

Comment:
Not really for this PR,, but can we dynamically evaluate the terminal width? I'm thinking, what if people resize while we're spinning.